README.md

```diff
@@ -10,13 +10,13 @@ docker run -it -p 127.0.0.1:8080:8080 -v "$PWD:/home/coder/project" codercom/cod
 ```
 
 - **Consistent environment:** Code on your Chromebook, tablet, and laptop with a
-  consistent dev environment. develop more easily for Linux if you have a
-  Windows or Mac, and pick up where you left off when switching workstations.
+  consistent dev environment. Develop more easily for Linux if you have a
```

Comment:
Develop more easily reads weird... let's change it to:

Develop on a linux machine and pickup where you left off from any device with a web browser.

Reply:
Kk